### PR TITLE
LUN-242: Header text could use more space

### PR DESCRIPTION
**Pull request by klinzo**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: src/navigation/Navigator.tsx [hunk 1]
```diff
@@ -37,11 +37,15 @@ export const styles = StyleSheet.create({
     borderBottomWidth: 1
   },
   headerRightContainer: {
-    paddingRight: 15,
-    flexShrink: 1
+    paddingHorizontal: 15,
+    maxWidth: 60
```
Comment (f1sh1918) [inline src/navigation/Navigator.tsx:41]: setting a maxWidth will unfortunately not do the trick thats why spent so much time on it.
The right container has to stay also flexible. Thats why i worked with proportion or you use flex-basis

**Review by f1sh1918 — CHANGES_REQUESTED** [review]
Tested on android. Right header container is too small in some exercises
You could add in `ExerciseHeader.tsx` a separate style, so Navigation Header will not be affected only Exercise Header:
` headerRightContainerStyle: {maxWidth:100, paddingRight:15}`

image

File: src/navigation/Navigator.tsx (hunk 1 above)
Comment (f1sh1918) [reply src/navigation/Navigator.tsx:41]: yes because this container gonna be used in different cases

File: src/components/ExerciseHeader.tsx [hunk 2]
```diff
@@ -32,6 +32,7 @@ interface ExerciseHeaderProps {
 
 const ExerciseHeader = ({ navigation, route, currentWord, numberOfWords }: ExerciseHeaderProps): JSX.Element => {
   const [isModalVisible, setIsModalVisible] = useState(false)
+  const progressText = numberOfWords !== 0 ? `${currentWord + 1} ${labels.general.header.of} ${numberOfWords}` : ''
```
Comment (klinzo) [inline src/components/ExerciseHeader.tsx:35]: Is this change fine @SteffiStuffel ?

**Review by f1sh1918 — APPROVED** [review]
Tested on android emulator and ios native. nice!

**Review by SteffiStuffel — APPROVED** [review]
Tested on browserstack android and ios.

File: src/components/ExerciseHeader.tsx (hunk 2 above)
Comment (klinzo) [reply src/components/ExerciseHeader.tsx:35]: May have been wanted :)

**Pull request merged**